### ISSUE TITLE
Prettier doesn't integrate well with 0.8.13 solidity

### ISSUE THREAD

**Issue by JuliusBrain** [prettier-solidity/prettier-plugin-solidity#673]
When using newest format for marking assembly block as memory safe, prettier throws following error `Error: extraneous input '(' expecting {'{', StringLiteralFragment}`, leaving the file unformatted.

Code block it fails at

```
assembly ("memory-safe") {
  ...
}
```

I've been also using compiler options to let the prettier use the newest solc version, it didn't help.
```
"options": {
        "compiler": "0.6.3"
      }
```

**Comment by mattiaerre** [comment]
thanks for reporting this issue @JuliusBrain

**Comment by pcaversaccio** [comment]
can confirm - you can reproduce this bug via eg. my contract here: https://github.com/pcaversaccio/create-util/blob/main/contracts/Create.sol. It currently breaks my CI lint pipeline, unfortunately. Adding `// prettier-ignore` also doesn't help either.

**Comment by Janther** [comment]
This has been addressed in #685 and will be part of the next release
